docs/data/data-grid/editing/CustomEditComponent.js

```diff
@@ -31,6 +31,10 @@ function RatingEditInputCell(props) {
     }
```

Comment:
Did you consider calling `preProcessEditCellProps` on mount, so that there's no need for `ensurePreProcessEditCellPropsRanOnce`?

Reply:
Yes, but I wanted to reproduce the same behavior of react-final-form. If I call it on mount, the field will appear invalid even if the user has not entered anything. See https://github.com/mui/mui-x/issues/5009#issuecomment-1139169542